tests/kafkatest/tests/core/consume_bench_test.py

```diff
@@ -68,12 +67,21 @@ def produce_messages(self, topics, max_messages=10000):
         self.logger.debug("Produce workload finished")
 
     @cluster(num_nodes=10)
-    @matrix(topics=[["consume_bench_topic[0-5]"]], metadata_quorum=quorum.all_non_upgrade) # topic subscription
-    @matrix(topics=[["consume_bench_topic[0-5]:[0-4]"]], metadata_quorum=quorum.all_non_upgrade)  # manual topic assignment
-    def test_consume_bench(self, topics, metadata_quorum=quorum.zk):
+    @matrix(
+        topics=[
+            ["consume_bench_topic[0-5]"],
+            ["consume_bench_topic[0-5]:[0-4]"]
+        ],
+        metadata_quorum=quorum.all_non_upgrade,
+        use_new_coordinator=[True, False]
+    ) # topic subscription
```

Comment:
You probably shouldn't skip zk + new coordinator test, just dont run it in the first place.  We dont really want to polute the report with a bunch of skips (which are normally reserved for broken test that we intend to fix later historicly).

```suggestion
    @matrix(
        topics=[
            ["consume_bench_topic[0-5]"],
            ["consume_bench_topic[0-5]:[0-4]"]
        ],
        metadata_quorum=metadata_quorum=[quorum.isolated_kraft],
        use_new_coordinator=[True, False]
    )
    @matrix(
        topics=[
            ["consume_bench_topic[0-5]"],
            ["consume_bench_topic[0-5]:[0-4]"]
        ],
        metadata_quorum=[quorum.zk],
        use_new_coordinator=[False]
    )  # topic subscription
```
as annoying as that is, prob for the best.

Reply:
I am +1 for not polluting the report. However, it is a bit annoying that we have to change all the tests... I was considering whether we should just add an extra "profile" to `quorum.all_non_upgrade` instead. Is there any downside to this approach?

Reply:
id argue we shouldn't tie configurations to eachother.  if you wanted to do that, then that would effect a lot of tests immediately.  That would be the easiest way to do that however now the configs are tied to eachother and you'd have to idividually support each permutation of using the config (for instance now you want to add colocated craft tests with new coordinator, or any other flag in the future that will be kraft exclusive which i imagine might be many). 

If you want to run with this globaly (all tests) Id argue the correct approach is to just use the `session_context.globals` (the issue is that kraft doesn't use this so kinda just stuck between a rock and a hard place, unless we run with the skip descriptor that is already there on all tests as well).

Reply:
I think the biggest object to that proposition i have is it isn't sustainable to any other kraft specific configs that might have to be added in the future.

Reply:
Oh I wasn't aware that Skips show up in the report, I thought it'd just actually skip the combination. I'll change it to what you suggested and remove the custom decorator.

Reply:
I also agree with Ian, I would definitely not want to tie these two together, just makes everything more confusing and messy, especially since this is temporary, we'd have to carefully decouple it again in a couple months. We can change it to a global config once we stop supporting zk.

Reply:
Yeah, I agree that relying on the `quorum.all_non_upgrade` is not perfect. The configuration that we are talking about here is temporary and will be removed in AK 4.0 (~Q2 24). At this time, the old group coordinator will be removed and KRaft will run by default with the new one. At the same time, we will remove ZK as well. Therefore, this flag won't be needed at all in a couple of months. We also want all the KRaft tests to run with the new group coordinator until we reach this goal.

@imcdo Knowing this, does it change your point of view or do you still believe that it is not good at all?

Reply:
I think if its only temporary its probably best as suggested as it would be easiest to role back and has the least complexity.  Yeah its a bit repetitive, but it wont get in the way of anyone trying to add configs in the next 2.5 quarters.  If we need a full test analysis of the feature, we can use the ducktape globals, and just live with the zk test results.  (and just not run them nightly, just when we want to validate).

Reply:
Though no strong opinion either way.  Just throwing in my 2 cents.